project-base/app/tests/App/Functional/Model/Transport/TransportPriceCalculationTest.php

```diff
@@ -51,7 +52,7 @@ public function testCalculatePrice(int $cartTotalWeight, int $expectedMoneyAmoun
 
         $expectedTransportPriceWithoutVat = $this->priceConverter->convertPriceWithoutVatToDomainDefaultCurrencyPrice(
             Money::create($expectedMoneyAmountWithoutVat),
-            $this->currencyFacade->getDomainDefaultCurrencyByDomainId(Domain::FIRST_DOMAIN_ID),
+            $this->getReference(CurrencyDataFixture::CURRENCY_CZK),
```

Comment:
Why CZK when we test first domain?

Reply:
This is currency of entering price so the converter knows correct rates for converting.